README.md

```diff
@@ -1,2 +1,5 @@
 # megafuel-go-sdk
-Golang SDK for the eoa-paymaster clients
+
+This is Golang SDK for the eoa-paymaster clients, it provides a simple way to interact with the eoa-paymaster.
```

Comment:
please provide a install sdk guide and a simple example about how to use the sdk

Reply:
fixed